src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(../src/images/star.svg);
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  margin-left: 4px;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../src/images/star-active.svg);
+}
```

Comment:
Be careful with the use of descendant combinator in selectors like `.stars--1 .stars__star:nth-child(-n + 1)`. These can be less efficient than class-based selectors and could potentially lead to performance issues on large documents. Consider refactoring to use a class that can be toggled on the `stars__star` elements directly.